schemas/events.v1.schema.json

```diff
@@ -1380,6 +1388,29 @@
         }
       }
     },
+    "ThreadChain": {
+      "title": "sentry_thread_chain",
+      "description": " One or multiple threads.",
+      "type": "object",
+      "required": [
+        "values"
+      ],
```

Comment:
@untitaker I know that Relay accepts the `threads` as an object instead of wrapped by the `values` array, so not sure if this is applicable here. I added an example for that and it fails because requires the `values`.
So what's the option here? removing the `required`?

Reply:
which example fails? is `"threads": {}` something we see in prod? if so, yes, remove required

Reply:
Nope, I made up, because of:

> An event may contain one or more threads in an attribute named threads.

It does not mention the `values` at all, so I will keep it as required as I can only reproduce this right now